frontend/src/components/home-page/content-tree.scss

```diff
@@ -1,5 +1,6 @@
 .content-tree {
```

Comment:
The styling for the home page needs to be fixed, particularly for wider screens. I don't think it's worth blocking the merge though for that.

Reply:
this is discrimination cuz my screen isn't wide :<